app/src/androidTest/java/ch/epfl/skysync/components/ConfirmFlightUITest.kt

```diff
@@ -201,25 +202,42 @@ class ConfirmFlightUITest {
   @Test
   fun confirmButtonWorksWhenDismiss() = runTest {
     composeTestRule.onNodeWithTag("AlertDialog").assertIsNotDisplayed()
+
+    composeTestRule.onNodeWithTag("LazyList").performScrollToNode(hasText("Enter Remark"))
+    composeTestRule.onNodeWithText("Confirm").assertDoesNotExist()
+
+    val setTime = composeTestRule.onAllNodesWithText("Set Time")
+    for (i in 0 until setTime.fetchSemanticsNodes().size) {
+      setTime[i].performClick()
+    }
```

Comment:
Shouldn't you scroll to those elements first before performing an action on them?

Reply:
It is not the best way to do it but `composeTestRule.onNodeWithTag("LazyList").performScrollToNode(hasText("Enter Remark")` ensures that all Set Times are displayed